ocaml/typing/ctype.ml

```diff
@@ -2677,6 +2677,11 @@ let rec expands_to_datatype env ty =
       end
   | _ -> false
 
+let equivalent_in_classic_mode l1 l2 =
+  l1 = l2 || (match l1, l2 with
+  | (Nolabel | Labelled _), (Nolabel | Labelled _) -> true
```

Comment:
Two `Labelled` args with different labels are also equivalent in classic mode.

Reply:
Yes, that's covered by the first case in the `match`

Reply:
And you are very right (thanks). I should remember the difference between commas and vertical bars.